cmd/buildctl/build/util.go

```diff
@@ -0,0 +1,32 @@
+package build
+
+import (
+	"errors"
```

Comment:
`pkg/errors`

Reply:
Done in https://github.com/moby/buildkit/pull/2707/commits/6ac606c9b135222934dda2b55e19ac9820c02b7c